Opcodes/mp3in.c

```diff
@@ -860,1622 +854,22 @@ static int32_t sprocess3(CSOUND *csound, DATASPACE *p)
 
 }
 
-#ifdef ANDROID
```

Comment:
I see a lot of code here removed for Android. Could you comment on this @vlazzarini so I can understand what's going on here?

Reply:
they need to go. They've already gone in other branches.